lib/database.js

```diff
@@ -2,7 +2,7 @@
 const fs = require('fs');
 const path = require('path');
 const util = require('./util');
-const CPPDatabase = require('../build/better_sqlite3.node').Database;
+const CPPDatabase = require('bindings')('better_sqlite3.node').Database;
```

Comment:
The reason I used the copy command was to remove the `bindings` dependency. I'm trying to have as few dependencies as possible. Is there any good reason why you changed that?

Reply:
The bindings dependency looks for the built .node file in all of the locations in which an addon has been compiled and loads it.  There are a few issues that people have submitted where it could not find ../build/better_sqlite3.node.  This would fix those issues.  I can remove the bindings dependency and look for the .node file in both Releases and Debug if you would prefer, because I  believe that having a copy of the .node file in the build directory is confusing.  Whereas, if you have one built in the Releases directory, it will use that, and if one is built in the Debug directory it will use that one.  Please let me know what you would prefer and I can do that.

Reply:
I don't see how the `bindings` module (or your hand-written solution) could fix the problem of people having those issues. In fact, `better-sqlite3` used to use `bindings`, and people still reported the same issue. The `copies` command is a cross-platform way of moving the binary to a reliable location, using node-gyp's `<(PRODUCT_DIR)` built-in variable, so we don't need guessing logic to find it. The reports of people not finding `../build/better_sqlite3.node` is probably related to their own systems, such as a lack of file permissions, etc.

Reply:
Ok. I guess I will find a way to change it back and still support using prebuild.

Reply:
Ok. I guess I will find a way to change it back and still support using prebuild.

Reply:
Oh I see. You're not using `bindings` to fix people's error, you're  using it because prebuild requires the binaries to be in their original location?

Reply:
Yes.  As far as I could tell I had to leave the binaries where they were for prebuild to bundle them correctly.

Reply:
Ahhhh. Well that's a much better reason to use `bindings`, in my opinion. I suppose it's not worth delaying this PR any longer. Go ahead and add back the dependency then.

Reply:
Added back.